### ISSUE TITLE
module问题

### ISSUE THREAD

**Issue by fanjing136** [vacancy/Synchronized-BatchNorm-PyTorch#39]
运行test报错：No module named 'models.networks.sync_batchnorm'，已经查过normalization.py和generator.py文件，只发现，在整个工程里只有这两个文件中提到models.networks.sync_batchnorm，但并没有接口或被调用？本人版本也在要求之内，确实想不出来是哪里出的问题了。请教了。
